src/clusterfuzz/_internal/cron/triage.py

```diff
@@ -298,6 +298,30 @@ def _file_issue(testcase, issue_tracker, throttler):
   return filed
 
 
+def _emmit_untriaged_testcase_age_metric(critical_tasks_completed: bool,
+                                         testcase: data_types.Testcase):
+  """Emmits a metric to track age of untriaged testcases."""
+  if critical_tasks_completed:
+    return
+  if not testcase.timestamp:
+    return
+  current_time = datetime.datetime.utcnow()
+  # testcase.timestamp is mutated in analyze task.
+  # taking the minimum between these two timestamps
+  # avoids the need to backfill creation_timestamp
+  testcase_creation_time = testcase.timestamp
+  if testcase.creation_timestamp:
```

Comment:
does this `if` exist purely for historical testcases?  

Instead of having this `if` here forever, I'd recommend just adding a `get_created_time()` (or similarly named) method to `Testcase` to perform this logic, with some comments explaining this.